src/resonanceReconstruction/rmatrix/src/makeReichMooreChannelData.hpp

```diff
@@ -124,12 +124,24 @@ makeReichMooreChannelData(
     auto rwFission2 = wFission2 | ranges::view::transform( toOtherWidth );
 
     // see how many channels we have
+    bool bElastic = not isAllZero( wElastic );
```

Comment:
This certainly works, but I don't like the naming conventions that led to `bElastic` and `cElastic` as variables.  I'd much rather see `elasticPopulated` or `hasElastic`, etc.

Reply:
Good point, this used to be my old naming convention: b for boolean. I changed it to has... since it is clearer.